mitplan-backend/src/server.ts

```diff
@@ -286,12 +281,7 @@ const startServer = async () => {
             id: defaultSheetId,
             name: 'Sheet 1',
             assignmentEvents: {},
-            encounter: {
-              events: [],
-              name: 'Default Encounter',
-              id: 'default',
-              fightLength: 600 // 10 minutes default fight length
-            },
+            encounterId: 'Default',
```

Comment:
encounterId should match something defined in the frontend. in this case you can put "default" or "rashok" I don't really care

Reply:
Default is correct here. The upper and lowercase screwed me up for quite awhile. It will crash using 'default'

Reply:
It must be indexing by the name of the typescript files in each encounter subfolder then, we should look at that later